features.json

```diff
@@ -6,56 +6,106 @@
+			"description": "Custom Text Format Annotations",
+			"url": "https://github.com/WebAssembly/annotations/blob/main/proposals/annotations/Overview.md",
+			"phase": 5
+		},
 		"customPageSizes": {
-			"description": "Custom page sizes",
+			"description": "Custom Page Sizes",
 			"url": "https://github.com/WebAssembly/custom-page-sizes/blob/main/proposals/custom-page-sizes/Overview.md",
 			"phase": 1
 		},
+		"esmIntegration": {
+			"description": "ESM integration",
+			"url": "https://github.com/WebAssembly/esm-integration",
+			"phase": 3
+		},
 		"exceptions": {
 			"description": "Exception handling",
 			"url": "https://github.com/WebAssembly/exception-handling/blob/master/proposals/exception-handling/Exceptions.md",
-			"phase": 3
+			"phase": 4
 		},
 		"extendedConst": {
-			"description": "Extended constant expressions",
+			"description": "Extended Constant Expressions",
 			"url": "https://github.com/WebAssembly/extended-const/blob/master/proposals/extended-const/Overview.md",
-			"phase": 4
+			"phase": 5
 		},
 		"gc": {
 			"description": "Garbage collection",
 			"url":"https://github.com/WebAssembly/gc",
-			"phase": 4
+			"phase": 5
+		},
+		"instrumentAndTracingTechnology": {
+			"description": "Instrument and Tracing Technology",
+			"url": "https://github.com/WebAssembly/instrument-tracing/blob/main/proposals/instrument-tracing/Overview.md",
+			"phase": 2
 		},
 		"jspi": {
 			"description": "JS Promise Integration",
 			"url":"https://github.com/WebAssembly/js-promise-integration",
 			"phase": 3
 		},
+		"jsStringBuiltins": {
+			"description": "JS String Builtins",
+			"url": "https://github.com/WebAssembly/js-string-builtins/blob/main/proposals/js-string-builtins/Overview.md",
+			"phase": 3
+		},
+		"halfPrecision": {
+			"description": "Half Precision",
+			"url": "https://github.com/WebAssembly/half-precision/blob/main/proposals/half-precision/Overview.md",
+			"phase": 1
+		},
 		"memory64": {
 			"description": "Memory64",
 			"url": "https://github.com/WebAssembly/memory64/blob/master/proposals/memory64/Overview.md",
 			"phase": 3
 		},
+		"memoryControl": {
+			"description": "Memory control",
+			"url": "https://github.com/WebAssembly/memory-control/blob/main/proposals/memory-control/Overview.md",
+			"phase": 1
+		},
 		"multiMemory": {
 			"description": "Multiple memories",
 			"url": "https://github.com/WebAssembly/multi-memory/blob/master/proposals/multi-memory/Overview.md",
-			"phase": 4
+			"phase": 5
 		},
 		"multiValue": {
 			"description": "Multi-value",
 			"url": "https://github.com/WebAssembly/spec/blob/master/proposals/multi-value/Overview.md",
-			"phase": 4
+			"phase": 5
 		},
 		"mutableGlobals": {
-			"description": "Mutable globals",
+			"description": "Import/Export of Mutable Globals",
 			"url": "https://github.com/WebAssembly/mutable-global/blob/master/proposals/mutable-global/Overview.md",
 			"phase": 5
 		},
+		"referenceTypedStrings": {
```

Comment:
stringref is implemented in V8, but it's not under active development anymore, and might disappear. I'm not sure I would want to include it here.